pytorch_lightning/trainer/trainer.py

```diff
@@ -964,34 +964,6 @@ def _run(self, model: "pl.LightningModule") -> Optional[Union[_EVALUATE_OUTPUT,
         self._call_configure_sharded_model()  # allow user to setup in model sharded environment
         self.accelerator.setup(self)
 
-        # ----------------------------
-        # INSPECT THE CORE LOOPS
-        # ----------------------------
-        fr"""
```

Comment:
shall this be a normal comment block?

Reply:
this has to stay please!

Reply:
I think so too, but the question s why it is a string not as a comment?

Reply:
because having it as an fstrings allows clicking to inspect the function definitions